[CI] Execution failed for task ':qa:mixed-cluster:v8.8.0#mixedClusterTest'

### CI Link

https://gradle-enterprise.elastic.co/s/x6ic2txirq4m4

### Repro line

n/a

### Does it reproduce?

Didn't try

### Applicable branches

8.8

### Failure history

_No response_

### Failure excerpt

Many networking error, but the first failure seemed to be related to remote transport validation, and an array length being unexpectedly long, perhaps a format change?

```
Caused by: java.lang.IllegalStateException: failure when sending a join validation request from [{v8.8.0-3}{TtjGlnzSTFeqNN2Z50UIQg}{u0sbUikVSpGkNs8vKOhqfg}{v8.8.0-3}{127.0.0.1}{127.0.0.1:43665}{cdfhilmrstw}{8.8.0}] to [{v8.8.0-0}{LlAaCjHISU-9qkHXvuJmbQ}{INlUgbJWS7q13B9UI7hK7A}{v8.8.0-0}{127.0.0.1}{127.0.0.1:46169}{cdfhilmrstw}{8.8.1}] |  
-- | --
  | »  	at org.elasticsearch.cluster.coordination.Coordinator.lambda$sendJoinValidate$13(Coordinator.java:730) |  
...
  | »  Caused by: org.elasticsearch.transport.RemoteTransportException: [v8.8.0-0][127.0.0.1:46169][internal:cluster/coordination/join/validate] |  
  | »  Caused by: java.lang.IllegalStateException: array length must be <= to 2147483631 but was: 2147483647
```

--- a/qa/mixed-cluster/build.gradle
+++ b/qa/mixed-cluster/build.gradle
@@ -38,6 +38,7 @@ BuildParams.bwcVersions.withWireCompatible { bwcVersion, baseName ->
       setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
       setting 'xpack.security.enabled', 'false'
       requiresFeature 'es.index_mode_feature_flag_registered', Version.fromString("8.0.0")
+      requiresFeature 'es.dlm_feature_flag_enabled', Version.fromString("8.8.0")
     }
 
     tasks.register("${baseName}#mixedClusterTest", StandaloneRestIntegTestTask) {

--- a/x-pack/qa/mixed-tier-cluster/build.gradle
+++ b/x-pack/qa/mixed-tier-cluster/build.gradle
@@ -29,6 +29,7 @@ BuildParams.bwcVersions.withWireCompatible(v -> v.onOrAfter("7.9.0") &&
       nodes."${baseName}-1".setting 'node.roles', '["data_content", "data_hot"]'
     }
     nodes."${baseName}-2".setting 'node.roles', '["master"]'
+    requiresFeature 'es.dlm_feature_flag_enabled', Version.fromString("8.8.0")
   }
 
   tasks.register("${baseName}#mixedClusterTest", StandaloneRestIntegTestTask) {

--- a/x-pack/qa/mixed-tier-cluster/build.gradle
+++ b/x-pack/qa/mixed-tier-cluster/build.gradle
@@ -1,6 +1,7 @@
 apply plugin: 'elasticsearch.legacy-java-rest-test'
 apply plugin: 'elasticsearch.bwc-test'
 
+import org.elasticsearch.gradle.Version
 import org.elasticsearch.gradle.VersionProperties
 import org.elasticsearch.gradle.internal.info.BuildParams
 import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask

